src/Models/VerbEvent.php

```diff
@@ -30,6 +28,16 @@ class VerbEvent extends Model
 
     protected ?Metadata $meta = null;
 
+    public function setTable($table): void
+    {
+        $this->table = $this->getTable();
+    }
```

Comment:
Is there any issue with `setTable()` not actually doing anything?

Reply:
@inxilpro yes. When `getTable()` is run, it returns the original table name `verb_events` instead of `sys_verb_events` in the tests. Doing this achieves the desired outcome of customizing the table name.

If you happen to know of a better solution, I'm open to making any necessary adjustments. 🤓

Reply:
@inxilpro so I definitely didn't see the obvious answer right in front of my face 🤦🏻‍♂️. I called the `setTable()` method in the tests and now everything is working as expected!